.cascade-code/Chapman.edu/_cascade/blocks/html/law_footer.html

```diff
@@ -0,0 +1,186 @@
+<!--
```

Comment:
I forgot to check the spacing. I'll fix that right now.

Reply:
Is this the production version of this file (plus your changes)? Doesn't look like it.

Best practices are to use production as the master version: https://wimops.chapman.edu/wiki/Cascade_Best_Practices#Version_Control_2

Reply:
I went into cascade and copied the XML from `Chapman.edu/_cascade/blocks/html/Law Footer`, since it didn't already exist within this repo. Was that the wrong version?

Reply:
Oh no that's right, I was looking at something different